src/compiler/diagnosticMessages.json

```diff
@@ -1040,7 +1040,7 @@
         "category": "Error",
         "code": 2306
     },
-    "Cannot find module '{0}'.": {
+    "Cannot find module '{0}' or cannot find corresponding type declarations for this module.": {

```

Comment:
@DanielRosenwasser  Shouldn't this be? `Cannot find module '{0}' or its corresponding type declarations.`

Reply:
I like that one better since it's a lot cleaner without giving up details 👍